packages/dd-trace/src/debugger/devtools_client/queue.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+class JSONQueue {
```

Comment:
nit: maybe JSONBuffer is better as this is not really a queue API

Reply:
Yeah I couldn't decide if I should call it a buffer or a queue... was considering both names. In Node.js there's a `Buffer` class which is different, so the term is a bit overloaded in Node.js. But I'll consider changing it... you're probably right

Reply:
I've renamed it from `Queue` to `JSONBuffer` now 👍